### DISCUSSION TITLE
show the progresses

### DISCUSSION THREAD

**Discussion by shabane** [shahriaarrr/Tasky#11] [Ideas]
یه فیچر دیگه که واقعا می‌تونه کمک کنه اینه که progress bar تحت ترمینال نشون بدی
دو نوع پروگرس نیازه 
- یکی برای اینکه چه تعداد تسک انجام شده بین کل تسک ها باقی مونده `[########__________]`
- دوم اینکه چقدر مونده برای رسیدن به اولین ددلاین، این می‌توی به صورت یه زمانی باشه البته 
```bash
[_____#####]
20 Minute left of 22 Hour
```

جریانش این هست که باید تسک ها رو زمان هاشون رو محاسبه کنه، مثلا اگه یه تسک از تاریخ فلان تا تاریخ فیلان وقت داشته، این محاسبه می‌کنه و میگه از 12 ساعت مثلا دیگه 2 ساعت مونده

همین فیچر رو با فلگ --all بده که همه تسک ها رو برات تایم هاشون رو حساب کنه.

**Comment by shahriaarrr** [comment]
ببین برای ددلاین که میگی خب عملا توی این نسخه های الان Tasky صرفا برای نوشتن لیست و تیک زدنشونه

ولی برای اینکه چه میزان تسکی مونده خب توی جداول قرار داده شده تعدادش.اگه براش بخوایم پراگرس بار بذاریم این پراگرس باز برای هر تسک مثلا چندتا هشتک قایل بشه؟ چجوری اینو محاسبه کنه؟

**Reply by shabane** [reply]
> ببین برای ددلاین که میگی خب عملا توی این نسخه های الان Tasky صرفا برای نوشتن لیست و تیک زدنشونه
> 
> ولی برای اینکه چه میزان تسکی مونده خب توی جداول قرار داده شده تعدادش.اگه براش بخوایم پراگرس بار بذاریم این پراگرس باز برای هر تسک مثلا چندتا هشتک قایل بشه؟ چجوری اینو محاسبه کنه؟



نه با درصد حساب می‌کنه، مثلا شما کلا 7 تا تسک داری، و 3 تاشو انجام دادی، اینطوری محاسبه می‌شه:

(3/7)*100
که می‌شه 42%

و حالا اگه همه 7 تا رو انجام بده میشه 
(7/7)*100

که می‌شه 100%

**Comment by shahriaarrr** [comment]
@shabane
